Assets/Scripts/Visuals/UiBuilder/Atom.cs

```diff
@@ -0,0 +1,79 @@
+using System.Collections.Generic;
+using JetBrains.Annotations;
+using UnityEngine;
+
+namespace Assets.Scripts.Visuals.UiBuilder
+{
+    public class AtomGameObject
+    {
+        public GameObject gameObject;
+        public UiBuilder.AtomType atomType;
+        public int height = -1;
+        public int position = -1;
+    }
+
+
+    public abstract class Atom
+    {
+        public UiBuilder uiBuilder;
+        public AtomGameObject gameObject;
+
+        protected Atom(UiBuilder uiBuilder)
+        {
+            this.uiBuilder = uiBuilder;
+        }
+
+        /// <summary>
+        /// Update the Atom given new Data
```

Comment:
Change comment wording. "Update the atom with the given data" or something similar